firebase_connector/models/firebase_notification.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Nicolas Badoux <n.badoux@hotmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models, fields
+from odoo.tools import config
+from odoo.exceptions import UserError
+from datetime import datetime
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import firebase_admin
+    from firebase_admin import credentials
+    from firebase_admin import messaging
+except ImportError as e:
+    _logger.error("Please install the PIP package firebase_admin")
+
+
+class FirebaseNotification(models.Model):
+    """
+    The represent a device that we can send push notification to. It is
+    characterized by the Firebase token and an optional partner.
+    It provides function to send messages to the device.
+    """
+
+    _name = 'firebase.notification'
+    _description = 'Notification to send to Firebase Cloud Messaging'
+
+    partner_ids = fields.Many2many('res.partner')
+    title = fields.Char(required=True)
+    body = fields.Char(required=True)
+    send_date = fields.Datetime()
+    sent = fields.Boolean(default=False, readonly=True)
```

Comment:
Default is already to False by default, so it's not needed.
```suggestion
    sent = fields.Boolean(readonly=True)
```